src/utils.js

```diff
@@ -176,6 +176,8 @@ export function iterateDataKey (key, callback) {
 }
 
 export function isFeatureBroken (args, feature) {
+    if (feature === 'clickToLoad') feature = 'clickToPlay'
```

Comment:
Can you just rename the feature? Theres a few places we check feature name that would be broken otherwise.

Reply:
This name is used in the config, etc.  Its not a trivial change to update it unfortunately, this was the most surgical fix that actually worked across the board.

Reply:
Can't we just rename click-to-load.js to click-to-play.js nothing else depends on this naming after it moves here no?

Reply:
Testing the following worked for me:

```diff
diff --git a/src/content-scope-features.js b/src/content-scope-features.js
index 86b13f8..4ba4aec 100644
--- a/src/content-scope-features.js
+++ b/src/content-scope-features.js
@@ -34,7 +34,7 @@ export async function load () {
         'fingerprintingScreenSize',
         'fingerprintingTemporaryStorage',
         'navigatorInterface',
-        'clickToLoad'
+        'clickToPlay'
     ]
 
     for (const featureName of featureNames) {
diff --git a/src/utils.js b/src/utils.js
index a2aa148..841b61e 100644
--- a/src/utils.js
+++ b/src/utils.js
@@ -176,8 +176,6 @@ export function iterateDataKey (key, callback) {
 }
 
 export function isFeatureBroken (args, feature) {
-    if (feature === 'clickToLoad') feature = 'clickToPlay'
-
     return isWindowsSpecificFeature(feature)
         ? !args.site.enabledFeatures.includes(feature)
         : args.site.isBroken || args.site.allowlisted || !args.site.enabledFeatures.includes(feature)
```

and rename src/features/click-to-load.js -> src/features/click-to-play.js

Reply:
Well yeah except we should move to calling it clickToLoad, also the new config is clickToLoad.  I don't really care TBH, we can just keep using clickToPlay.  We'll have to update the new config to revert this too but it can wait.

Reply:
Ok reverted to using CTP, rebased, rebuild, tests pass locally.  Not sure why its complaining in CI, I just built again and there's nothing to push?

Reply:
It's CTP in the config: https://github.com/duckduckgo/privacy-configuration/blob/main/features/click-to-play.json

It's failing the clean tree test, run npm build again and check in any assets changes. If that doesn't work check your node/npm version to match CI.

Reply:
> Well yeah except we should move to calling it clickToLoad, also the new config is clickToLoad. I don't really care TBH, we can just keep using clickToPlay.

Which new config? The tds stuff?
Yeah I don't care what it is; if we're migrating the remote config name then let's do that automated etc and avoid these feature names in a function; there's other util functions that take a config name that would need changing.

Reply:
Yeah lets just punt on this for now, its not blocking.  There was much discussion to call it "click to load" vs "play" so was trying to be consistent, but it can wait.